modules/repository-url/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:18.04
+
+RUN apt-get update -qqy
+RUN apt-get install -qqy openjdk-11-jre-headless
```

Comment:
why not just use an existing java base image?

Reply:
I followed the pattern we use for the other docker based fixtures (e.g. gcs) which uses the same images. Which image would you suggest to base this on?